cmd/server/customers.go

```diff
@@ -105,18 +106,20 @@ func respondWithCustomer(logger log.Logger, w http.ResponseWriter, customerID st
 // TODO(adam): What GDPR implications does this information have? IIRC if any EU citizen uses
 // this software we have to fully comply.
 type customerRequest struct {
-	FirstName  string              `json:"firstName"`
-	MiddleName string              `json:"middleName"`
-	LastName   string              `json:"lastName"`
-	NickName   string              `json:"nickName"`
-	Suffix     string              `json:"suffix"`
-	Type       client.CustomerType `json:"type"`
-	BirthDate  model.YYYYMMDD      `json:"birthDate"`
-	Email      string              `json:"email"`
-	SSN        string              `json:"SSN"`
-	Phones     []phone             `json:"phones"`
-	Addresses  []address           `json:"addresses"`
-	Metadata   map[string]string   `json:"metadata"`
+	CustomerID string                `json:"-"`
```

Comment:
Added this `CustomerID` so I can reuse this request object in `updateCustomer`. 

This field will be set by the path var in the URL.

Reply:
Sounds good. Overtime I'd like use to use `pkg/client.CustomerRequest` models, but I know Birthdate and others make that harder to do.